code/game/objects/items/props/helmetgarb.dm

```diff
@@ -496,6 +496,26 @@
 	desc = "The USCM had its funding pulled for these when it became apparent that not every deployed enlisted was wearing a helmet 24/7; much to the bafflement of UA High Command."
 	icon_state = "helmet_gasmask"
 
+/obj/item/prop/helmetgarb/helmet_gasmask/on_enter_storage(obj/item/storage/internal/helmet_internal_inventory)
+	..()
+	var/obj/item/clothing/head/helmet/helmet_item = helmet_internal_inventory.master_object
```

Comment:
```suggestion
/obj/item/prop/helmetgarb/helmet_gasmask/on_enter_storage(obj/item/storage/internal/helmet_internal_inventory)
	..()
	if(!istype(helmet_internal_inventory))
		return
	var/obj/item/clothing/head/helmet/helmet_item = helmet_internal_inventory.master_object
```
It appears this is an incorrect assumption the argument passed will be specifically an `obj/item/storage/internal`:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/c1dffcbf-a8d1-4ccf-84ab-bbe6b679a120)
So my review https://github.com/cmss13-devs/cmss13/pull/5083#discussion_r1414834147 ended up to be incorrect. I missed that the argument was changed to be `obj/item/storage/internal` instead of just `obj/item/storage`